collector/filesystem_linux.go

```diff
@@ -219,15 +217,26 @@ func parseFilesystemLabels(r io.Reader) ([]filesystemLabels, error) {
 		parts[4] = strings.ReplaceAll(parts[4], "\\011", "\t")
 
 		filesystems = append(filesystems, filesystemLabels{
-			device:      parts[m+3],
-			mountPoint:  rootfsStripPrefix(parts[4]),
-			fsType:      parts[m+2],
-			options:     parts[5],
-			major:       fmt.Sprint(major),
-			minor:       fmt.Sprint(minor),
-			deviceError: "",
+			device:       parts[m+3],
+			mountPoint:   rootfsStripPrefix(parts[4]),
+			fsType:       parts[m+2],
+			mountOptions: parts[5],
+			superOptions: parts[10],
+			major:        fmt.Sprint(major),
+			minor:        fmt.Sprint(minor),
```

Comment:
`strconv.Itoa` is about 3x faster than `fmt.Sprint` for simple use cases like this.